Mail/smtp.php

```diff
@@ -185,7 +204,8 @@ class Mail_smtp extends Mail {
      * passed in. It looks for the following parameters:
      *     host        The server to connect to. Defaults to localhost.
      *     port        The port to connect to. Defaults to 25.
-     *     auth        SMTP authentication.  Defaults to none.
+     *     auth        SMTP authentication. Defaults to none.
+     *     starttls    Should STARTTLS connection be used? Defaults to false. PEAR/Net_SMTP >= 1.10.0 required.
```

Comment:
The default is not `false` anymore.

Reply:
Hawkeye ;-) I´m sorry. I have pushed another commit with "No default".